docs/specs/xref.yml

```
@@ -2162,3 +2162,69 @@ outputs:
   .errors.log: |
     {"message_severity":"warning","code":"missing-attribute","message":"Missing required attribute: 'name'.","file":"docs/TOC.md"}
     {"message_severity":"warning","code":"xref-not-found","message":"Cross reference not found: 'b'.","file":"docs/TOC.md"}
+---
+# files to be redirected must be excluded from xref map
+noSingleFile: true
+repos:
+  https://docs.com/redirected-uid-excluded-from-xrefa#master:
+    - files:
+        docfx.yml:
+        _themes/ContentTemplate/schemas/TestData.schema.json: |
+          {
+            "renderType": "component",
+            "properties": {
+              "uid": { "contentType": "uid" },
+              "fullName": { "type": "string" },
+              "name": { "type": "string" },
+              "title": { "type": "string" }
+            },
+            "xrefProperties": ["title","name","fullName"]
+          }
+        a.yml: |
+          ### YamlMime:TestData
+          uid: a
+          title: Title a.yml
+          name: name a
+          fullName: fullName a
+        b.yml: |
+          ### YamlMime:TestData
+          uid: b
+          title: Title b
+          name: name b
+          fullName: fullName b
+        redirections.yml: |
+          redirections:
+            a.yml: /absolute/path2
+outputs: 
+  .xrefmap.json: | 
+    {
+      "references":[
+        {
+          "uid": "b",
+          "title": "Title b",
+          "name": "name b",
+          "fullName": "fullName b"
+        }
+      ]
+    }
+  b.json: | 
+    {
+      "_xrefmap":
+        {
+          "b":
+          {
+            "uid":"b",
+            "href":"b.json",
+            "name":"name b",
+            "schemaType":"TestData",
+            "title":"Title b",
+            }
+         },
+       "uid":"b",
+       "fullName":"fullName b",
+       "name":"name b",
+       "schema":"TestData",
+       "title":"Title b",
+     }
+  .errors.log: | 
+    {"message_severity":"warning","code":"redirected-file-not-removed","message":"Redirected file 'a.yml' are still in the repo. After adding a file to the redirection JSON file, you must delete the original file from the repo.","line":0,"end_line":0,"column":0,"end_column":0,"log_item_type":"user"}
```

Remove properties with default values.
```suggestion
    {"message_severity":"warning","code":"redirected-file-not-removed","message":"Redirected file 'a.yml' are still in the repo. After adding a file to the redirection JSON file, you must delete the original file from the repo."}
```